docs/website/docs/tutorial/rest-api.md

```diff
@@ -0,0 +1,320 @@
+                    "params": {
+                        "limit": 1000,
+                    },
+                },
+            },
+            "resources": [
+                "pokemon",
+                "berry",
+                "location",
+            ],
+        }
+    )
+
+    ...
+
+    load_info = pipeline.run(pokemon_source)
+    print(load_info)
+```
+
+Here what's happening in the code:
+
+1. With `dlt.pipeline()` we define a new pipeline named `rest_api_pokemon` with DuckDB as the destination and `rest_api_data` as the dataset name.
+2. The `rest_api_source()` function creates a new REST API source object.
+3. We pass this source object to the `pipeline.run()` method to start the pipeline execution. Inside the `run()` method, dlt will fetch data from the API and load it into the DuckDB database.
+4. The `print(load_info)` outputs the pipeline execution details to the console.
+
+Let's break down the configuration of the REST API source. It consists of three main parts: `client`, `resource_defaults`, and `resources`.
+
+```py
+config: RESTAPIConfig = {
+    "client": {
+        ...
+    },
+    "resource_defaults": {
+        ...
+    },
+    "resources": [
+        ...
+    ],
+}
+```
+
+- The `client` configuration is used to connect to the web server and authenticate if necessary. For our simple example, we only need to specify the `base_url` of the API: `https://pokeapi.co/api/v2/`.
+- The `resource_defaults` configuration allows you to set default parameters for all resources. Normally you would set common parameters here, such as pagination limits. In our Pokemon API example, we set the `limit` parameter to 1000 for all resources to retrieve more data in a single request and reduce the number of HTTP API calls.
+- The `resources` list contains the names of the resources you want to load from the API. REST API will use some conventions to determine the endpoint URL based on the resource name. For example, the resource name `pokemon` will be translated to the endpoint URL `https://pokeapi.co/api/v2/pokemon`.
+
+:::note
+### Pagination
+You may have noticed that we didn't specify any pagination configuration in the `rest_api_source()` function. That's because for REST APIs that follow best practices, dlt can automatically detect and handle pagination. Read more about [configuring pagination](../dlt-ecosystem/verified-sources/rest_api.md#pagination) in the REST API source documentation.
+:::
+
+## Appending, replacing, and merging loaded data
+
+Try running the pipeline again with `python rest_api_pipeline.py`. You will notice that
+all the tables have data duplicated. This happens because by default, dlt appends the data to the destination table. It is very useful, for example, when you have daily data updates and you want to ingest them. In dlt you can control how the data is loaded into the destination table by setting the `write_disposition` parameter in the resource configuration. The possible values are:
```

Comment:
>It is very useful, for example, when you have daily data updates and you want to ingest them

Is it? Aren't you just reloading the new data and the old data and then you have duplicated data anyways? Just wondering whether it makes sense to keep this sentence in here.

Reply:
True, carried this sentence over from another docs as an explanation why append is needed. Let's remove the sentence.